sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -192,13 +171,6 @@ int main()
 
   printf("\n\n*********************************\nDELETED Key: \n %s", response_buffer);
```

Comment:
WOW! This code got a lot better!
Much better for customers!

Reply:
its keyvault anyway. A dream only a few had in the past XD